hasjob/views/index.py

```diff
@@ -819,3 +819,18 @@ def logoimage(domain, hashid):
 @app.route('/search')
 def search():
     return redirect(url_for('index', **request.args))
+
+
+@app.route('/offline')
```

Comment:
Can we be careful about adding new top-level routes? This merits a discussion. See #145.

Reply:
Changed it to `/api/1/template/offline`